rustls/src/server/tls12.rs

```diff
@@ -640,8 +664,8 @@ impl hs::State for ExpectCertificateVerify {
             let handshake_msgs = self.transcript.take_handshake_buf();
             let certs = &self.client_cert;
 
-            cx.config
-                .get_verifier()
+            self.config
+                .verifier
```

Comment:
Why not `get_verifier()`?

Reply:
Changed it back. Will add another changes to get rid of `get_verifier()` to another PR.